packages/react/src/components/Masthead/MastheadSearchSuggestion.js

```diff
@@ -47,7 +50,7 @@ const MastheadSearchSuggestion = ({
       className={classNames('container-class', {
         ['container-highlight-class']: isHighlighted,
```

Comment:
Should these have prefixes, or are they something external?

Reply:
This isn't something Juan is asking for yet in analytics so we should probably let him make requests as he needs.

Reply:
Not sure I understand. Why is Juan dictating our `classNames`?

Reply:
He isn't. I mean we don't need to prefix them with stable selectors yet.

Reply:
I see the confusion. Sorry, I'm not talking stable selectors here `data-autoid`. My question was about the `className` on this line and the one above. `container-highlight-class` Is it supposed to be prefixed liked `#{prefix}--container-highlight-class`?

Reply:
Ah I got you. Yeah we can (should) prefix that.